Makefile

```diff
@@ -24,6 +24,11 @@ $(NAME).tar.gz:
 	mv ocaml packages
 	mv upstream-extra packages
 
+licenses:
+	opam list --required-by "xs-toolstack" --recursive --short \
+	  | xargs opam info -f name:,license: | tr -d '"' \
+	  | awk '/^name:/ { name = $$2 } /^license:/ { license = $$2; printf("%s: %s\n",name, license); }'
+
```

Comment:
Did you intend to keep the command in both files?

Reply:
Sorry - missed adding the updated Makefile